docs/manual.rst

```diff
@@ -479,11 +482,26 @@ The file format is a bencoded dictionary containing the following fields:
 |                          | for, in case it is a v2 or v1+v2-hybrid torrent. This is a   |
 |                          | 32 byte SHA-256 hash of the info section of the torrent.     |
 +--------------------------+--------------------------------------------------------------+
-| ``pieces``               | A string with piece flags, one character per piece.          |
+| ``pieces``               | **deprecated**. This field is no longer written, but can     |
+|                          | can still be read.                                           |
+|                          | A string with piece flags, one character per piece.          |
 |                          | Bit 1 means we have that piece.                              |
 |                          | Bit 2 means we have verified that this piece is correct.     |
 |                          | This only applies when the torrent is in seed_mode.          |
 +--------------------------+--------------------------------------------------------------+
+| ``pieces``               | A string where each bit indicates whether we have the piece  |
+|                          | or not.                                                      |
+|                          |                                                              |
+|                          | In version 1 of this file format, this was a string with one |
+|                          | byte for every piece. Bit 0 meant we had the piece, bit 1    |
+|                          | meant the piece was verified (for seed-mode lazy checking)   |
++--------------------------+--------------------------------------------------------------+
+| ``verified``             | A string where each bit indicates whether the piece has been |
+|                          | verified to be correct. This is used in seed-mode, where     |
+|                          | pieces are checked lazily.                                   |
+|                          |                                                              |
+|                          | This field did not exist in version 1 of this file format.   |
```

Comment:
Is it correct? Or is there another "verified" field in version 1?

Reply:
it is correct. in version 1, the `pieces` field used a full byte per piece. one of those bits meant "we have the piece" and another bit meant "the piece has been verified (in seed-mode". The remaining 6 bits were wasted space.

Reply:
but there's another `verified` field under the v2 merkle trees too, but that's separate